docker/README.md

```diff
@@ -5,46 +5,117 @@ Thats right folks onedrive is now dockerized ;)
 
 ## Usage instructions
+
+### 0. Install the docker under your own platform's instructions
+
+### 1. Pull the image
+
 ```bash
 docker pull driveone/onedrive
 ```
+
 **NOTE:** SELinux context needs to be configured or disabled for Docker, to be able to write to OneDrive host directory.
 
-1.  Run or update onedrive container
+### 2. Prepare required stuff
+
+Onedrive needs two volumes. One of them is the config volume. 
+
+If you dont't need an extra config file, You can create a docker volume:
+
 ```bash
-# Update onedriveDir with correct existing OneDrive directory path
-onedriveDir="${HOME}/OneDrive"
+docker volume create onedrive_conf;
+```
 
-firstRun='-d'
-docker pull driveone/onedrive
-docker inspect onedrive_conf > /dev/null || { docker volume create onedrive_conf; firstRun='-it'; }
-docker inspect onedrive > /dev/null && docker rm -f onedrive
-docker run $firstRun --restart unless-stopped --name onedrive -v onedrive_conf:/onedrive/conf -v "${onedriveDir}:/onedrive/data" driveone/onedrive
+This will create a docker volume labeled 'onedrive_conf', which we will use it later.
+
+The second one is your data folder that needs to sync with. Keep in mind that:
+
+-   The owner of the folder must not be root
+
+-   The owner have permission to its parent directory  
+    (because onedrive will try to setup a monitor for the sync folder).
+
+### 3. First run
+
+Onedrive also needs to be authorized with your account.  
+This is done by running docker in interactive mode. 
+
+**make sure to change onedriveDir to your own.**
+
+```bash
+onedriveDir="${HOME}/OneDrive"
+docker run -it --restart unless-stopped --name onedrive -v onedrive_conf:/onedrive/conf -v "${onedriveDir}:/onedrive/data" driveone/onedrive
 ```
-## Poweruser section
-1.  Check if monitor service is running
+
+-   You will be asked to open a specific link using your web browser 
+-   login into your Microsoft Account and give the application the permission  
+-   After giving the permission, you will be redirected to a blank page.  
+-   Copy the URI of the blank page into the application.
+
+If your onedrive is working as expected, you can detach from the container with Ctrl+p, Ctrl+q.
+
+### 4. Status, stop, and restart
+
+Check if monitor service is running
+
 ```bash
 docker ps -f name=onedrive
 ```
-2.  Show monitor run logs
+
+Show monitor run logs
+
 ```bash
 docker logs onedrive
 ```
-3.  Stop running monitor
+
+Stop running monitor
+
 ```bash
 docker stop onedrive
 ```
-4.  Resume monitor
+
+Resume monitor
+
 ```bash
 docker start onedrive
 ```
-5.  Unregister onedrive monitor
+
+Unregister onedrive monitor
+
 ```bash
 docker rm -f onedrive
 ```
+
+### 5. Edit the config
+
+Onedrive should run in default configuration, but however you can change your configuration.  
+
+First download the default config from [here](https://raw.githubusercontent.com/abraunegg/onedrive/master/config)  
+Then put it into your onedrive_conf volume: `/var/lib/docker/volumes/onedrive_conf/_data`
```

Comment:
This is *probably* depending on the way docker was installed. Here on my Debian system this is the correct location, but I wouldn't bet on it.